README.md

```diff
@@ -289,20 +292,38 @@ Below you can find the list of methods that can be used as expectations:
 
 > Note: if you use such methods as expectations they will not count as expectations, in a test report, for example, but if they fail a clear message will be shown to ease understanding why the test has failed. In the end, test reports are usually useful in cases where tests have failed and we need to understand why. If all tests are green there is nothing to worry about, at least not if they were well implemented.
 
-### Example of a test failure when using such methods as expectations
+### Examples of a test failure when using such methods as expectations
```

Comment:
Actually it should be Examples of test failures ...

Reply:
Done.

Reply:
Nope. It should be test failure**s**